content/body/screen-reader-only-text.php

```diff
@@ -32,7 +32,7 @@
       visually hidden headings to complete a page's outline</a> from <a
       href="https://www.accessibility-developer-guide.com/">Accessibility
       Developer Guide</a> talks about this in depth.</li>
-    <li>Text read in your screen reader's reading mode (i.e. using it to read the page, not for reading out the interactive elements that have focus), screen reader only text is guaranteed to work.  ARIA labels might not be read in your screen readers reading mode.</li>
+    <li>Text read in your screen reader's reading mode (i.e. using it to read the page, not for reading out the interactive elements that have focus), screen reader only text is guaranteed to work.  ARIA labels might not be read in your screen reader reading mode.</li>
```

Comment:
I think that this makes the last sentence more clear:

```suggestion
    <li>Text read in your screen reader's reading mode (i.e. using it to read the page, not for reading out the interactive elements that have focus), screen reader only text is guaranteed to work.  ARIA labels might not be read in the reading mode of your screen reader.</li>
```